src/components/scenes/SwapProcessingScene.tsx

```diff
@@ -33,7 +37,7 @@ export interface SwapProcessingParams {
 
 type Props = SwapTabSceneProps<'swapProcessing'>
 
-export function SwapProcessingScene(props: Props) {
+export const SwapProcessingScene: React.FC<Props> = (props: Props) => {
```

Comment:
You don't need `(props: Props) =>` anymore. Just do `props =>` now, since we already give the type on the left.

Reply:
Please delete the redundant `: Props`.